hyperstart.go

```diff
@@ -406,10 +406,10 @@ func (h *hyper) exec(pod *Pod, c Container, process Process, cmd Cmd) error {
 	}
 
 	if _, err := h.proxy.sendCmd(proxyCmd); err != nil {
-		return err
+		return guestProcess{}, err
 	}
 
-	return nil
+	return guestProcess{}, nil
```

Comment:
I'm not clear why hyperstart always returns an empty `guestProcess`?

Reply:
Because the shim does not need this information. Also because no process ID is returned from the protocol.